fiat-roles/src/main/groovy/com/netflix/spinnaker/fiat/permissions/DefaultPermissionsResolver.java

```diff
@@ -67,8 +57,8 @@
     if (!anonymousEnabled) {
       return Optional.empty();
     }
-    return Optional.of(getUserPermission(AnonymousUserConfig.ANONYMOUS_USERNAME,
-                                         new ArrayList<>(0) /* groups */));
+    return getUserPermission(AnonymousUserConfig.ANONYMOUS_USERNAME,
+                             new ArrayList<>(0) /* groups */);
```

Comment:
It's a feature of Java 7 - the diamond operator is used by the compiler to infer the type.
 
http://www.javaworld.com/article/2074080/core-java/jdk-7--the-diamond-operator.html